requirements.txt

```diff
@@ -9,21 +9,23 @@ Jinja2==2.7.2
 MarkupSafe==0.21
 SQLAlchemy==0.9.4
 Werkzeug==0.9.4
-amqp==1.4.5
+amqp==2.1.4
 anyjson==0.3.3
-bcrypt==1.0.2
-billiard==3.3.0.17
-celery==3.1.11
-cffi==0.8.2
+bcrypt==3.1.3
+billiard==3.5.0.2
+celery==4.0.2
+cffi==1.10.0
 cssmin==0.2.0
 hiredis==0.1.3
 ipython<5
 itsdangerous==0.24
-kombu==3.0.15
+kombu==4.0.2
 pycparser==2.10
 pytz==2014.2
 redis==2.9.1
-requests==2.2.1
+requests==2.13.0
 speaklater==1.3
-webassets==0.9
+webassets==0.12.1
 wsgiref==0.1.2
+psycopg2==2.7.1
+pyOpenSSL==16.2.0
```

Comment:
Yes, but then should it be installed separately? I can't run [this](https://github.com/mvidalgarcia/cephalopod/blob/d4e6aebcd310b7295701677bee8386b2da8addfe/cephalopod/manage.py#L18) without pyOpenSSL